dwh/query_lib_big_query.py

```diff
@@ -0,0 +1,325 @@
+"""Module that implements Query Engine for BigQuery."""
+import typing as tp
+
+import pandas as pd
+
+from base import EncounterConstraints, ObsConstraints, PatientQuery
+
+
+_DATE_VALUE_SEPARATOR = ","
+
+try:
+    from google.cloud import bigquery
+except ImportError:
+    pass  # not all set up need to have bigquery libraries installed
+
+
+def _build_in_list_with_quotes(values: tp.Iterable[tp.Any]):
```

Comment:
Can't we just use List[str] instead of `tp.Iterable[tp.Any]`?
Please also add the returned value type.